nautobot_golden_config/utilities/db_management.py

```diff
@@ -0,0 +1,15 @@
+"""Functions to manage DB related tasks."""
+from django.db import connections
+
+
+def close_threaded_db_connections(func):
+    """Decorator that clears idle DB connections in thread."""
+
+    def inner(*args, **kwargs):
+        """Inner function."""
+        func(*args, **kwargs)
+
+        for c in connections.all():
+            c.connection.close()
```

Comment:
```suggestion
        for connection in connections.all():
            connection.connection.close()
```
Not tested, suggestion just to satisfy pylint.

Reply:
Just wondering if this needs to be implemented somewhere else.  Multiple other plugins in the future will use the same nornir nautobot plugins in the backend potentially a solution that can go in over there?

Also this will only run on the worker?

Reply:
@jeffkala I do see your point but this fix is addressing an issue (stale DB connections) that is caused on the plugin level before calling `dispatcher`. Do you think we could address this concern with some documentation enhancements to draw developers' attention to the fact that threads do not close DB connections?